tests/django_s3_storage_test/tests.py

```diff
@@ -15,20 +14,29 @@
 from django.utils import timezone
 from django.utils.timezone import is_naive, make_naive, utc
 
-from django_s3_storage.storage import S3Storage, StaticS3Storage
+from django_s3_storage.storage import Endpoints, S3Storage, StorageIsReadOnly
 
 
-class TestS3Storage(SimpleTestCase):
+def helper_old_to_new_name(name) -> str:
+    return f's3://1928-django-s3-storage-test/{name}'
+
 
-    def tearDown(self):
-        # clean up the dir
-        for entry in default_storage.listdir(""):
-            default_storage.delete("/".join(entry))
+class TestS3Storage(SimpleTestCase):
+    # def tearDown(self):
+    #     # clean up the dir
+    #     for entry in default_storage.listdir(helper_old_to_new_name("")):
+    #         print(entry)
+    #         # default_storage.delete("/".join(entry))
 
     # Helpers.
 
     @contextmanager
-    def save_file(self, name="foo.txt", content=b"foo", storage=default_storage):
+    def save_file(
+        self,
+        name=f"s3://1928-django-s3-storage-test/foo.txt",
```

Comment:
Unnecessary f-string

```suggestion
        name="s3://1928-django-s3-storage-test/foo.txt",
```